independent-publisher-connectors/Microsoft 365 Service Health/apiDefinition.swagger.json

```diff
@@ -0,0 +1,303 @@
+                            "featureGroup": {
+                              "type": "string",
+                              "description": "featureGroup"
+                            },
+                            "isResolved": {
+                              "type": "boolean",
+                              "description": "isResolved"
+                            },
+                            "details": {
+                              "type": "array",
+                              "items": {},
+                              "description": "details"
+                            },
+                            "posts": {
+                              "type": "array",
+                              "items": {
+                                "type": "object",
+                                "properties": {
+                                  "createdDateTime": {
+                                    "type": "string",
+                                    "description": "createdDateTime"
+                                  },
+                                  "postType": {
+                                    "type": "string",
+                                    "description": "postType"
+                                  },
+                                  "description": {
+                                    "type": "object",
+                                    "properties": {
+                                      "contentType": {
+                                        "type": "string",
+                                        "description": "contentType"
+                                      },
+                                      "content": {
+                                        "type": "string",
+                                        "description": "content"
+                                      }
+                                    },
+                                    "description": "description"
+                                  }
+                                }
+                              },
+                              "description": "posts"
+                            }
+                          }
+                        },
+                        "description": "issues"
+                      }
+                    }
+                  },
+                  "description": "value"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get Health",
+        "description": "Get Health",
+        "operationId": "Health",
+        "x-ms-visibility": "important",
+        "parameters": [
+          {
+            "name": "$expand",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "x-ms-visibility": "advanced",
+            "enum": [
+              "",
+              "issues"
+            ]
+          }
+        ]
+      }
+    },
+    "/beta/admin/serviceAnnouncement/healthOverviews": {},
```

Comment:
@techienickb Please do not add empty definitions in the swagger as they would not add any value to your connector.